### ISSUE TITLE
CI broken on master since December 4th

### ISSUE THREAD

**Issue by ctaggart** [Azure/azure-rest-api-specs#8324]
It appears that CI has been failing on the master branch since December 4th when "Enable SDK Automation V2 in CI" was merged. #7548 @NullMDR @akning-ms 

![image](https://user-images.githubusercontent.com/80104/73943340-59307700-48cf-11ea-9047-b72c874e02f7.png)

Can you please figure out how to make CI on master green again?

**Comment by NullMDR** [comment]
The failing of Advocado is a known issue @ruowan . For SDK Automation itself its high failing rate is currently known and expected. Which CI failure do you want to call out @ctaggart ?

**Comment by ctaggart** [comment]
Why is it known and expected to have a high failing fate of CI failures on the master branch? If it is just the Advocado tests that are flaky. Why not disable them to make CI green? 

On #8296, I'm trying to make CI pass. It is failing on LintDiff and I can't figure out why or reproduce it locally. I began to wonder if it was even passing on master and then I noticed master CI has been broken for two months. Having a broken CI for the master branch in my experience, is usually really bad. I understand, this situation may be different, but I don't currently know why.
